src/Microsoft.Security.Utilities.Core/IdentifiableSecrets.cs

```diff
@@ -226,7 +226,8 @@ public static ulong ComputeHisV1ChecksumSeed(string versionedKeyKind)
             if (testChar == null)
             {
                 using var generator = RandomNumberGenerator.Create();
-                generator.GetBytes(keyBytes, 0, (int)keyLengthInBytes);
+
+                generator.GetBytes(keyBytes);
```

Comment:
In all the usages of `GetBytes`, the byte array length is precisely equal to the number of bytes required. So the 3 argument `GetBytes` API is not strictly required.